openage/convert/driver.py

```diff
@@ -279,7 +282,7 @@ def convert_media(args):
             # do the dir "renaming"
             if dirname == "gamedata" and filepath.suffix == ".slp":
                 output_dir = "graphics"
-            elif dirname == "gamedata" and filepath.suffix == ".wav":
+            elif dirname == "interface" and filepath.suffix == ".wav":
```

Comment:
My aoe installation (aoe II from the collectors edition with conquerors expansion) has no `wav` files in `gamedata.drs`, but in `interfaces.drs`. I assume, this was a simple error from TheJJ. Please check you're installation.